config/webhook/manifests.yaml

```diff
@@ -0,0 +1,26 @@
+
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: validating-webhook-configuration
+webhooks:
+- clientConfig:
+    caBundle: Cg==
+    service:
+      name: webhook-service
+      namespace: system
+      path: /validate-infrastructure-cluster-x-k8s-io-v1alpha2-awsmachinetemplate
+  failurePolicy: Fail
+  name: vawsmachinetemplate.kb.io
```

Comment:
How was this name generated?

Reply:
kubebuilder does it this way

Reply:
Got it, was mostly curious as it reads funny